report/verification/mms/simple_temperature.md

```diff
@@ -0,0 +1,282 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.2
+kernelspec:
+  display_name: vv-festim-report-env
+  language: python
+  name: python3
+---
+
+# Simple diffusion with varying temperature
+
+```{tags} 2D, MMS
+```
+
+This is an extension of the simple MMS example with a non-homogenous temperature gradient.
```

Comment:
Could you add a clickable link to the case?

Reply:
got it :+1: